Cargo.toml

```diff
@@ -31,5 +31,8 @@ rand_chacha = "0.1"
 rand = "0.6"
 
 [features]
+default = ["std"]
 nightly = ["clear_on_drop/nightly"]
 debug-transcript = ["hex"]
+std = ["rand_core/std", "byteorder/std"]
+alloc = ["rand_core/alloc"]
```

Comment:
Does the `alloc` feature actually get used for anything? I don't think Merlin allocates, so if this feature is only being used to enable a corresponding feature in `rand_core`, I'm not sure we need it -- since Cargo features are additive, anything else that needs `rand_core/alloc` can enable it themselves.